lib/utils/fs/writeFile.js

```diff
@@ -1,16 +1,17 @@
 'use strict';
 
+const fse = require('./fse');
 const path = require('path');
+const jc = require('json-cycle');
 const YAML = require('js-yaml');
-const fse = require('./fse');
 
 function writeFile(filePath, conts) {
   let contents = conts || '';
 
   return fse.mkdirsAsync(path.dirname(filePath))
     .then(() => {
       if (filePath.indexOf('.json') !== -1 && typeof contents !== 'string') {
-        contents = JSON.stringify(contents, null, 2);
+        contents = jc.stringify(contents);
```

Comment:
FWIW, the 2nd and 3rd parameters are not recognized by `json-cycle` so it doesn't yet make sense to pass them

Reply:
resolving this pends upon the acceptance of https://github.com/valery-barysok/json-cycle/pull/4

Reply:
@erikerikson can't we just pass the values in for now (although they're ignored)? This way we could bump the package version number later on and it would work OOTB without the need to touch the code here again.